.github/workflows/trivy.yml

```diff
@@ -0,0 +1,53 @@
+name: Trivy Scanner
+on: workflow_dispatch
```

Comment:
This has manually been dispatched to ensure it works as expected.

Reply:
Success here: https://github.com/DataBiosphere/terra-data-catalog/actions/runs/3002284454